code/package.json

```diff
@@ -38,13 +38,15 @@
     "@testing-library/jest-dom": "^6.4.5",
     "@testing-library/react": "^15.0.7",
     "@testing-library/user-event": "^14.5.2",
+    "@types/dotenv-webpack": "^7.0.7",
     "@types/jest": "^29.5.12",
     "@types/react": "^18.3.3",
     "@types/react-dom": "^18.3.0",
     "babel-jest": "^29.7.0",
     "babel-loader": "^9.1.3",
     "babel-plugin-module-resolver": "^5.0.2",
     "css-loader": "^7.1.2",
+    "dotenv-webpack": "^8.1.0",
```

Comment:
this should be dev dependency.
How you will access environment variables in the cloud (in prod environment)? Most probably you won't have env.local there.

Reply:
`dotenv-webpack` and `@types/dotenv-webpack` are already in dev dependecies. Or am I missing something?